Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs

```diff
@@ -49,31 +50,27 @@ public ConsumerAppController(IConsumerAppRepository iConsumerAppRepository, User
         * @apiSuccessExample {json} Success-Response:
         * HTTP/1.1 200 OK 
         * {
-        *     "description":"retun true when added succesfully.If any problem in addconsumer mehtod or not added consumer apps so return false."
+        *     "description":"return true if succesfully consumer app has been added else return false."
```

Comment:
I have not change anything in ConsumerAppController. so you want I have change the ConsumerAppController apidoc.